ignite/services/plugin/plugin.go

```diff
@@ -76,6 +76,19 @@ func CollectEvents(ev events.Bus) Option {
 	}
 }
 
+// Execute starts and executes a plugin, then shutdowns it.
+func Execute(ctx context.Context, path string, args ...string) error {
```

Comment:
> Because this will be used internally in Ignite CLI for specific cases, I don't feel that we need to document this feature in the official documentation.

Should we put this in an internal package then? Given that we follow semver now, I'd be wary to introduce public API that is supposed to be used internally only.

Reply:
Good point, do we already have such a folder somewhere ?

Reply:
I don't think so we have any in /plugin, so plugin/internal would make sense.

Reply:
done here 2e4ab969bd362e2d6092f5731bfb6188152b1197